app/uk/gov/hmrc/disareturns/services/ReturnsSummaryService.scala

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2025 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.disareturns.services
+
+import uk.gov.hmrc.disareturns.models.common.Month.Month
+import uk.gov.hmrc.disareturns.models.summary.repository.SaveReturnsSummaryResult
+import uk.gov.hmrc.disareturns.repositories.{MonthlyReturnsSummaryRepository, ReturnMetadataRepository}
+
+import javax.inject.{Inject, Singleton}
+import scala.concurrent.{ExecutionContext, Future}
+
+@Singleton
+class ReturnsSummaryService @Inject() (
+  metadataRepo: ReturnMetadataRepository,
+  summaryRepo:  MonthlyReturnsSummaryRepository
+)(implicit ec:  ExecutionContext) {
+
+  def saveReturnsSummary(zRef: String, year: Int, month: Month, totalRecords: Int): Future[SaveReturnsSummaryResult] =
+    metadataRepo
+      .existsForZrefAndPeriod(zRef, year, month)
```

Comment:
I think we had a convo around this but I'm not sure we want this logic.
ReturnMetadataRepository has a ttl of 30 days. 
If for some reason the summary result has taken more than 30 days to come back to us. We would reject this summary result, where as really we would want to still store it.

If the return metadata wasn't found, we would need to try retrieve the boxId from PPNS to send off the notification off.